docassemble/mlhframework/data/questions/mlh_feedback.yml

```diff
@@ -6,11 +6,12 @@ include:
 ---
 metadata:
   title: User Feedback
-  exit url: https://da-dev.mplp.org/list
```

Comment:
That makes sense, I can make that change to the configs and delete that line.

I'm a bit torn but I **think** the main server pages make sense as a place for the default exitpage. The other obvious option would be MLH homepage. I think we'll often want to send people to MLH, but especially in this situation where we already have two tabs open, I think it might be less confusing to stay on the forms server.

This is mostly a note to self, but correct me if I'm wrong: I hadn't realized the Exit button deleted variables--need to keep an eye on that if we have Exit buttons later in interviews after people may have entered a bunch of info. I see we can [modify](https://docassemble.org/docs/initial.html#exit%20link) the exit behavior as well.

Reply:
hmm, if we just take the line out it jumps to courtformsonline instead, presumably because the AL feedbackform yaml _does_ specify an exit url.

The exit button actually doesn't appear on the prod server. Just the resume or start over buttons, so it's not currently a problem to leave it as is.